gulp-core-build/src/jsonUtilities/SchemaValidator.ts

```diff
@@ -45,16 +38,16 @@ export class SchemaValidator {
    */
   public static validate(data: Object, schema: Object, dataFilePath?: string): void {
     if (!this._schemaValidator.validate(data, schema)) {
-      const error: ISchemaValidatorResult = this._schemaValidator.getLastError();
```

Comment:
@nickpape-msft

NOTE: The point of Validator.getLastError() is to return an Error subclass that is suitable for throwing.  It is a wrapper that does not add any new information, so if you don't need that, it's better to call getLastErrors() instead.